### PR TITLE
Styled file inputs

### PR DESCRIPTION
**Pull request by romawht22**

### CODE REVIEW

**Review by naberalgamma — APPROVED** [review]

**Review by obozhko-dev — COMMENTED** [review]
1. In Profile page when I tap to "Edit profile" button , image moved to right side 
image

2. In "Create location" window I can add only one photo now. If i didn't mistake Andrey did functionality to download more than 1 photo. Сheck with him, please, you need to upload one photo or many?

File: frontend/src/components/CreateLocation/CreateLocation.tsx [hunk 1]
```diff
@@ -135,14 +134,30 @@ const CreateLocation = ({ coordinate }: Props) => {
...
-      />
+      <Box>
+        <label htmlFor="contained-button-file">
+          <HiddenInput
+            id="contained-button-file"
+            type="file"
+            onChange={e => {
+              handleFilesChange(e);
+              setlocationImageName(
+                (ref.current?.files && ref.current?.files[0].name) || ''
```
Comment (Yojuks) [inline frontend/src/components/CreateLocation/CreateLocation.tsx]: you should add items-alient: center to this block if you use flex

**Review by Yojuks — CHANGES_REQUESTED** [review]

File: frontend/src/components/Profile/ProfilePage.tsx [hunk 2]
```diff
@@ -127,21 +128,32 @@ export default function ProfilePage({
...
+                  src={
+                    (userImage && URL.createObjectURL(userImage)) ||
+                    userAvatar.data.imageUrl
+                  }
                 />
-                <Box sx={{ m: '2vh 0 2vh 14vh' }}>
-                  {t('profile.profilePage.uploadPhoto')}
-                </Box>
+                <Box>{t('profile.profilePage.uploadPhoto')}</Box>
                 <Box>
```
Comment (6mahdapihka9) [inline frontend/src/components/Profile/ProfilePage.tsx]: what is it for?

**Review by 6mahdapihka9 — COMMENTED** [review]
I think you should create the whole component that includes button and hiddenInput and takes props like title, onChange etc. as mui objects. That will make it more reusable and convenient

File: frontend/src/components/design/UploadInputProfilePage/index.tsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+          sx={{ m: '10px 0 10px 0' }}
+          variant="contained"
+          component="span"
+        >
+          {t('profile.profilePage.uploadPhoto')}
+        </Button>
+      </label>
+    </Box>
+  );
+};
```
Comment (Yojuks) [inline frontend/src/components/design/UploadInputProfilePage/index.tsx:37]: Screenshot 2022-05-30 at 14 02 49м
Screenshot 2022-05-30 at 14 08 24
text and button maybe should be by center)

**Review by Yojuks — APPROVED** [review]

**Review by 6mahdapihka9 — APPROVED** [review]
norm

**Pull request merged**